src/main/java/uk/gov/companieshouse/model/Attachment.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.companieshouse.model;
+
+import java.util.Objects;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class Attachment {
+
+    @NotBlank(message = "attachement id must not be null.")
+    private String id;
+
+    @NotBlank(message = "name must not be blank.")
+    private String name;
+
+    @NotBlank(message = "contentType must not be blank.")
+    private String contentType;
+
+    @NotNull()
+    private Integer size;
```

Comment:
What happens if the size is zero or a negative? We could improve the validation to compare against a max size - as we do on the front end

Reply:
Makes sense to have min value set as non zero.

Reply:
Can you add custom error messages for the two constraints above for consistency?